modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -45,34 +45,39 @@ To configure a Mule 4.x app in Studio 7.x:
 . Install Object Store Connector for Mule 4:
 .. Click the *Exchange* icon in the toolbar:
 +
-image::os-tutorial-studio7-exchange.png["Exchange icon in the Studio 7 toolbar"]
+image::os-tutorial-studio7-exchange.png["Studio 7.x window showing the Exchange icon in the toolbar"]
 .. Enter your Anypoint Platform username and password, if prompted.
 .. In Exchange, select *Provided by MuleSoft* and *Connectors*, and then search for Object Store Connector:
 +
-image::os-tutorial-exchange.png["Provided by MuleSoft option and search field in Exchange"]
+image::os-tutorial-exchange.png["Exchange window showing the Provided by MuleSoft option and search field"]
 .. Select *ObjectStore Connector - Mule 4*.
 .. Click *Add to project*:
 +
-image::os-tutorial-exchange-add-to-project.png["Add to project icon in Exchange"]
+image::os-tutorial-exchange-add-to-project.png["Exchange window showing the Add to project icon"]
 .. At the confirmation prompt, click *Proceed* and then click *OK*.
 . In the *Mule Palette* view, click *HTTP*, and then drag *Listener* to the Studio canvas:
 +
-image::os-tutorial-studio7-palette.png["HTTP Connector and Listener operations in the Mule Palette view"]
+image::os-tutorial-studio7-palette.png["Mule Palette view showing the HTTP Connector and Listener operations"]
++
+[calloutlist]
+.. HTTP Connector
+.. Listener operation
+
 . Click *ObjectStore* and then drag the *Store* operation to *Listener* on the canvas:
 +
-image::os-tutorial-studio7-flow.png["Store operation in the Studio 7 canvas"]
+image::os-tutorial-studio7-flow.png["Studio 6.x window showing the Store operation on the canvas"]
```

Comment:
Isn't Studio 6 deprecated?

Reply:
Studio 6.x is still available, but now I'm wondering if that image is correct 🤣

Reply:
Fixed.